docs/source/reference-io.rst

```diff
@@ -304,6 +304,8 @@ unfortunately that's not yet possible.
 
    .. automethod:: statistics
 
+.. autoclass:: DTLSChannelStatistics
```

Comment:
https://trio--2704.org.readthedocs.build/en/2704/reference-io.html#trio.DTLSChannel.statistics looks like we might want to de-duplicated these docs a bit; make the first a cross-reference to the newly-added second?

Reply:
Should do the same with the other *Statistics.